packages/client/components/auth/src/flows/Form.tsx

```diff
@@ -82,30 +92,32 @@ export function Fields(props: FieldProps) {
   return (
     <For each={props.fields}>
       {(field) => (
-        <FormGroup>
-          {field === "log-out" ? (
-            <label class={labelRow()}>
-              <Checkbox name="log-out" />
-              <Typography variant="label">
-                {fieldConfiguration["log-out"].name()}
-              </Typography>
-            </label>
-          ) : (
-            <>
-              <Typography variant="label">
-                {fieldConfiguration[field].name()}
-              </Typography>
-              <Input
-                required
-                {...fieldConfiguration[field]}
-                name={field}
-                placeholder={fieldConfiguration[field].placeholder()}
-                submissionTried={failedValidation()}
-                onInvalid={onInvalid}
-              />
-            </>
-          )}
-        </FormGroup>
+        <Show when={field != "invite" || inviteCodeNeeded}>
+          <FormGroup>
+            {field === "log-out" ? (
+              <label class={labelRow()}>
+                <Checkbox name="log-out" />
+                <Typography variant="label">
+                  {fieldConfiguration["log-out"].name()}
+                </Typography>
+              </label>
+            ) : (
+              <>
+                <Typography variant="label">
+                  {fieldConfiguration[field].name()}
+                </Typography>
+                <Input
+                  required
+                  {...fieldConfiguration[field]}
+                  name={field}
+                  placeholder={fieldConfiguration[field].placeholder()}
+                  submissionTried={failedValidation()}
+                  onInvalid={onInvalid}
+                />
+              </>
+            )}
+          </FormGroup>
+        </Show>
```

Comment:
Nit: I would rather use a Switch component here, rather than using ternaries.

Reply:
Do you mean the bit inside the FormGroup? That hasn't been changed (indentation was added) but I can change it up to use that

Reply:
Oh i see

I was reviewing the pr from mobile so it wasn't clear what exactly was changed

you can leave it as is for now

Reply:
I just changed it thought :weh:

Reply:
eh it's fine
it should do the same thing anyways :trollface:

Reply:
ok but is the new code good?

Reply:
LGTM